### PR TITLE
Add shortcut to site on showcase item

### PR DESCRIPTION
**Pull request by riencoertjens**

### CODE REVIEW

**Comment by riencoertjens** [comment]
#8098

File: www/src/views/showcase/showcase-list.js [hunk 1]
```diff
@@ -56,29 +55,27 @@ const ShowcaseList = ({ items, count }) => {
...
-                  </div>
-                )}
+                  )}
+                  <a
+                    href={node.main_url}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    css={{...styles.shortcutIcon}}
+                  >
+                    {` `}
```
Comment (fk) [inline www/src/views/showcase/showcase-list.js]: Nitpicky alert, but we should move that blank `{` `}` in between the two `href`s/only render it when we have a link to the GitHub source.

**Comment by fk** [comment]
@shannonbux This adds an "icon button" that allows to visit the showcase site without having to click through to the detail page:

image

I'm in favor of getting this in, what do you think?

**Comment by shannonbux** [comment]
oh yes, I support this! @fk let's do it!

**Pull request merged**